From aabca34878d5c4493c8ec984f16abea8e5f31845 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 4 Jul 2022 11:50:18 +0100
Subject: [PATCH] dtoverlays: Use PDAF variant for imx258 overlay

There are a number of variants of the imx258 sensor. The only
version that is currently made into a module for use with the Pi
is from Vision Components, and that supports PDAF and needs the
appropriate register configuration.

Update the compatible string accordingly.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/imx258-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/imx258-overlay.dts b/arch/arm/boot/dts/overlays/imx258-overlay.dts
index 7a8ed540438e..3cefd59663a0 100644
--- a/arch/arm/boot/dts/overlays/imx258-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx258-overlay.dts
@@ -48,7 +48,7 @@ __overlay__ {
 			status = "okay";
 
 			imx258: imx258@10 {
-				compatible = "sony,imx258";
+				compatible = "sony,imx258-pdaf";
 				reg = <0x10>;
 				status = "okay";
 
-- 
2.32.0

